taglib/mp4/mp4itemfactory.h

```diff
@@ -158,7 +158,7 @@ namespace TagLib {
       /*!
        * Destroys the frame factory.
        */
-      virtual ~ItemFactory();
+      ~ItemFactory();
```

Comment:
`ItemFactory` is meant to be subclassed and has virtual methods, so the destructor should be virtual too.